tools/site_and_regional/subset_data

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+"""
+This is a just top-level skeleton script that calls
+subset_data.py.
+The original code (subset_data.py) is located under
+python/ctsm folder.
+
+For full instructions on how to run the code and different options,
+please check python/ctsm/subset_data.py file.
+This script extracts domain files, surface dataset, and DATM files
+at either a single point or a region using the global dataset.
+
+To see all available options for single-point subsetting:
+    ./subset_data point --help
+To see all available options for region subsetting:
+    ./subset_data region --help
+
+To run this script the following packages are required:
+    - numpy
+    - xarray
+----------------------------------------------------------------
+To see all available options for single-point/regional subsetting:
+    ./subset_data --help
+"""
+
```

Comment:
They do need to have access to the right modules. So ncar_pylib is one way to do that. But, also note that module load conda is replacing ncar_pylib on cheyenne. But, @negin513 already has several notes about loading ncar_pylib. So for example the following note shows up when you actually run help...

```
|------------------------------------------------------------------|
|---------------------  Instructions  -----------------------------|
|------------------------------------------------------------------|
Instructions for running on Cheyenne/Casper:
load the following into your local environment
    module load python
    ncar_pylib
```

So I would call this done.

Reply:
Yes. as @ekluzek mentioned I have had the instructions on top of `subset_data.py` as the above. I've also added this to the top of `subset_data`. 

Another thing is that I am trying to write the third party packages that are needed for each python code so it would be easier for the users without any access to cheyenne and ncar_pylib to run it:
https://github.com/ESCOMP/CTSM/blob/13fec89d1b8f5ee2fcd3e406a66f3fa226ef085e/tools/site_and_regional/subset_data#L13-L30